jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/LogCleanupCommandTest.java

```diff
@@ -280,26 +280,27 @@ public void deleteAllLogsOlderThanTomorrowDifferentProcess() throws Exception {
         Assertions.assertThat(getNodeInstanceLogSize(HELLO_WORLD_ID)).isPositive();
     }
 
-    @Test(timeout = 10000)
+    @Test
     public void deleteAllLogsPage() {
-        CountDownAsyncJobListener countDownListener = new CountDownAsyncJobListener(1);
-        ((ExecutorServiceImpl) getExecutorService()).addAsyncJobListener(countDownListener);
         // Generate data
         KieSession kieSession = createKSession(HELLO_WORLD);
         startProcess(kieSession, HELLO_WORLD_ID, 3);
-
         // Verify presence of data
-        Assertions.assertThat(getProcessLogSize(HELLO_WORLD_ID)).isEqualTo(3);
-        Assertions.assertThat(getNodeInstanceLogSize(HELLO_WORLD_ID)).isPositive();
-
+        int processSize = getProcessLogSize(HELLO_WORLD_ID);
+        int nodeSize = getNodeInstanceLogSize(HELLO_WORLD_ID);
+        Assertions.assertThat(processSize).isEqualTo(3);
+        Assertions.assertThat(nodeSize).isPositive();
+        
+        CountDownAsyncJobListener countDownListener = new CountDownAsyncJobListener(processSize + nodeSize);
+        ((ExecutorServiceImpl) getExecutorService()).addAsyncJobListener(countDownListener);
+        
         // Schedule cleanup job
-        scheduleLogCleanup(false, false, false, null, null, HELLO_WORLD_ID, 1);
+        scheduleLogCleanup(false, false, true, null, null, HELLO_WORLD_ID, 1);
```

Comment:
not sure about this change? why are we skipping the ExecutorLogs?

Reply:
Thats to make the test predictable, we want to delete the nodes and the process logs, so the number of transactions, if recordsPerTransaction is 1,  is the sum of number of nodes plus the number of process (which has to be deleted in the latest transactions because they are used to resolve the other items to be deleted) 
At the end, the test is pretty complete, we are checking the boolean, the number of transactions and the items are really deleted.